pallets/staking/src/lib.rs

```diff
@@ -483,12 +487,25 @@ impl<AccountId> Default for RewardDestination<AccountId> {
 }
 
 /// Preference of what happens regarding validation.
-#[derive(PartialEq, Eq, Clone, Default, Encode, Decode, RuntimeDebug)]
+#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug)]
 pub struct ValidatorPrefs {
     /// Reward that validator takes up-front; only the rest is split between themselves and
     /// nominators.
     #[codec(compact)]
     pub commission: Perbill,
+    /// Whether or not this validator is accepting more nominations. If `true`, then no nominator
+    /// who is not already nominating this validator may nominate them. By default, validators
+    /// are accepting nominations.
+    pub blocked: bool,
```

Comment:
Doesn't this come with the migration?

Reply:
Fixed in https://github.com/PolymathNetwork/Polymesh/pull/1022/commits/80d670e47b68c0c93d1ad4c9dbb0179e69495eeb.